assets/finamp.desktop.m4

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=Finamp
+GenericName=Music Player
+Icon=finamp
+Exec=__INSTALL_PATH__/finamp
```

Comment:
is this a variable that needs to be changed? if so, a comment (if possible) would be nice.

Reply:
It does, it's meant to be templated with m4. I'll add a note to CONTRIBUTING.md, that should be ok.

Reply:
Note added, I think this should be enough for people packaging the app for their distribution.